package.json

```diff
@@ -19,6 +19,7 @@
     "locales": "npm run locales:setup && npm run locales.src",
     "xcode-export-locales": "rm -rf ./xcode/en.xcloc && xcodebuild -exportLocalizations -project ./xcode/Ghostery.xcodeproj -localizationPath ./xcode",
     "release": "./scripts/release.sh",
+    "udpate-scriptlets": "deno scripts/update-scriptlets.js --tagName 1.61.3b4 > ./src/rule_resources/scriptlets.js",
```

Comment:
If downloading scripts requires Deno, and we cannot avoid it - the file should be generated on the server and downloaded like Trackers Preview data. We can provide just what this script outputs.

Reply:
if the POC works then we can follow suggestion of @philipp-classen and move the scriptlets to a separate repository